### PR TITLE
Mongo metadata

### PR DESCRIPTION
**Pull request by depohmel**

### CODE REVIEW

File: go/src/metadata/mongo/mongo.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+		return models.ImageList{}, err
+	}
+	c := s.Conn.DB(dbName).C(collName)
+	imgs := models.ImageList{}
+	return imgs, c.Find(nil).All(&imgs)
+}
+
+// FindImages performs a select and returns list of images
+func (s *Mongo) FindImages(ctx context.Context, tags map[string]string) (models.ImageList, error) {
+	return models.ImageList{}, nil
```
Comment (ashmrtn) [inline go/src/metadata/mongo/mongo.go:40]: I assume this is going to be filled out in another patch?

Comment (depohmel) [reply go/src/metadata/mongo/mongo.go:40]: Not really, I thought about it as a search function... 
So I might add it, but it's a low priority for now.
